packages/turbo/src/VisitableView.tsx

```diff
@@ -169,6 +188,7 @@ const VisitableView = React.forwardRef<RefObject, React.PropsWithRef<Props>>(
 
     return (
       <>
+        {errorComponent}
         {loadingComponent}
```

Comment:
Thinking we'd probably want to only render one view here (loading should take precedence).

There's a similar pattern in [`react-native-webview`](https://github.com/react-native-webview/react-native-webview/blob/6960a1903c3c32e1e77be11e6370f1c384ed7ee4/src/WebView.ios.tsx#L140-L149C7)